upup/pkg/fi/cloudup/tagbuilder.go

```diff
@@ -23,6 +23,9 @@ func buildCloudupTags(cluster *api.Cluster) (map[string]struct{}, error) {
 		// external is based on kubenet
 		tags["_networking_kubenet"] = struct{}{}
 		tags["_networking_external"] = struct{}{}
+	} else if networking.CNI != nil {
+		// external is based on cni, weave, flannel, etc
+		tags["_networking_cni"] = struct{}{}
```

Comment:
Also answered in French. Again just modeling code done before. We can cleanup if you care to recommend a pattern.